docs/docs/the_aztec_network/guides/run_nodes/how_to_run_sequencer.md

```diff
@@ -26,29 +26,142 @@ tags:
 
 The Aztec sequencer node is critical infrastructure responsible for ordering transactions and producing blocks.
 
-When transactions enter the network, the sequencer node bundles them into blocks, checking various constraints such as gas limits, block size, and transaction validity. Before a block can be published, it must be validated by a committee of other sequencer nodes (validators in this context) who re-execute the transactions to verify their correctness. These validators attest to the block's validity by signing it, and once enough attestations are collected (two-thirds of the committee plus one), the sequencer can submit the block to L1.
+The sequencer node takes part in three key actions:
+1. Assemble unprocessed transactions and propose the next block
+2. Attest to correct execution of txs in the proposed block (if part of validator committee)
+3. Submit the successfully attested block to L1
+
+When transactions are sent to the Aztec network, sequencer nodes bundles them into blocks, checking various constraints such as gas limits, block size, and transaction validity. Before a block can be published, it must be validated by a committee of other sequencer nodes (validators in this context) who re-execute public transactions and verify private function proofs so they can attest to correct execution. These validators attest to the block's validity by signing it, and once enough attestations are collected (two-thirds of the committee plus one), the sequencer can submit the block to L1.
 
 The archiver component complements this process by maintaining historical chain data. It continuously monitors L1 for new blocks, processes them, and maintains a synchronized view of the chain state. This includes managing contract data, transaction logs, and L1-to-L2 messages, making it essential for network synchronization and data availability.
 
-## Prerequisites
+## Setup
+
+### Requirements
+
+A computer running Linux or MacOS with the following specifictions:
+- CPU: 8-cores
+- RAM: 16 GiB
+- Storage: 1 TB SSD
+
+A Network connection of at least 25 Mbps up/down.
+
+### Installation
 
-Before following this guide, make sure you:
+import { General, Fees } from '@site/src/components/Snippets/general_snippets';
+
+<General.InstallationInstructions />
+
+Now install the latest testnet version of aztec: `aztec-up alpha-testnet`
 
-- Have the `aztec` tool [installed](../../../developers/getting_started.md#install-the-sandbox)
-- Set up `docker` on your system. Refer to the [Docker installation guide](https://docs.docker.com/engine/install/).
-- You are using the correct version for the testnet by running `aztec-up alpha-testnet`
-- Are running a Linux or MacOS machine with access to a terminal
 
 Join the [Discord](https://discord.gg/aztec) to connect with the community and get help with your setup.
 
-## Requirements
 
-- Network: 25 Mbps up/down
-- CPU: 8-cores
-- RAM: 16 GiB
-- Storage: 1 TB SSD
+## Sequencer Quickstart
```

Comment:
Sorry 1 more thing here. I think you need to add a bit about port forwarding to this quickstart section. its a requirement for the node to work properly

Reply:
Have added it after the start command.
I read it was specific configurations that it would not work in. I had tried both with my node and it seemed to be performing the same.